src/nix/flake.cc

```diff
@@ -63,16 +66,38 @@ class FlakeCommand : virtual Args, public MixFlakeOptions
 
 struct CmdFlakeUpdate : FlakeCommand
 {
+private:
+
+    std::vector<std::string> passedInputs;
```

Comment:
`inputs` is already a pretty vague term, so let's give the reader a slight bit of extra context to understand what this variable is meant for :)

```suggestion
    std::vector<std::string> inputsToUpdate;
```

Reply:
(nit): Maybe also/alternatively make it a `std::vector<FlakeInput>` and parse it on the fly to give it a more meaningful type?

Reply:
Ah, very good point! I moved the parsing into the handler and place the inputPath directly into `lockFlags` now instead of collecting it first and then iterating over it again.